tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/pariah/body_size.tsx

```diff
@@ -0,0 +1,6 @@
+import { FeatureChoiced, FeatureDropdownInput } from "../../base";
```

Comment:
This file should be moved one folder up to `\features\character_preferences`, since it's still semi-moduar being inside `\features\character_preferences\pariah`.